app/tests/spec/views/sign_up.js

```diff
@@ -872,6 +873,53 @@ function (chai, $, sinon, p, View, Coppa, Session, AuthErrors, Metrics,
           });
       });
     });
+
+    describe('passwordCheck', function () {
+      beforeEach(function () {
+        view.passwordcheck = new PasswordCheck();
+        view.logError = function (err) {
```

Comment:
Instead of overwriting the function, use a [sinon spy](http://sinonjs.org/docs/#spies) and then in the tests use `calledWith`. Plenty of examples around to see how that works.